sdks/java/javadoc/dataflow-sdk-docs/package-list

```diff
@@ -1,5 +1,5 @@
 com.google.cloud.dataflow.sdk
-com.google.cloud.dataflow.sdk.annotations
+org.apache.beam.sdk.annotations
```

Comment:
Normally, I'd order this alphabetically. However, since all of these will become org.apache.beam, please leave as-is.